public/js/Histogram.js

```diff
@@ -0,0 +1,100 @@
+var d3 = require('d3');
+var _ = require('lodash');
+_.mixin(require('lodash-deep'));
+
+function Histogram(sampleData, element, featureNames) {
```

Comment:
No, JavaScript has no class :P It's a prototypical inheritance language. So these plots aren't defined as their own class, they are extensions of a base ``Object`` class.